extensions/libraries/redcore/translation/helper.php

```diff
@@ -789,4 +789,142 @@ public static  function setLanguage($language, $loadLanguage = true)
 
 		return false;
 	}
+
+	/**
+	 * Checks if the current page is a translatable form.
+	 *
+	 * @return void
+	 */
+	public static function isTranslatableForm()
+	{
+		$input = JFactory::getApplication()->input;
+		$option = $input->getString('option', '');
+		$view = $input->getString('view', '');
+		$layout = $input->getString('layout', '');
+		$task = $input->getString('task', '');
+
+		$translationTables = self::getInstalledTranslationTables();
+
+		if ($layout == 'edit' || $task == 'edit')
+		{
+			//Go through all installed translation tables
+			foreach ($translationTables as $tableKey => $translationTable)
+			{
+				if (!empty($translationTable->formLinks))
+				{
+					//Go through all form links
+					foreach ($translationTable->formLinks as $formLink)
+					{
+						$formLinks = explode('#', $formLink);
+
+						//Check whether the form link values matches the current page
+						if ($formLinks[0] == $option && $formLinks[1] == $view)
+						{
```

Comment:
after view there should be layout check which is by default "edit" but you should still check the layout here